LearningMissionLab/Resume.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace LearningMissionLab
+{
+    public class Resume
+    {
+        readonly Guid _resumeId;
+        string _firstName;
+        string _lastName;
+        DateTime _birthDate;
+        string _birthPlace;
+        ContactInfo _contactInfo;
+        List<WorkExperience> _workExperienceList;
+        List<Education> _educationList;
+        HashSet<Skill> _skillsHashSet;
+        HashSet<Language> _languageHashSet;
+        HashSet<string> _interestList;
+
```

Comment:
Add linkedIn and github links. You can add them as separate fields, or (it would be better) add a list of SocialNetwork objects

Reply:
Done.